src/renderers/shaders/ShaderChunk/shadowmap_pars_fragment.glsl

```diff
@@ -16,4 +16,124 @@
 
 	}
 
+	#if defined(POINT_LIGHT_SHADOWS)
+
+		float unpack1K ( vec4 color ) {
+		
+			const vec4 bitSh = vec4( 1.0 / (256.0 * 256.0 * 256.0), 1.0 / (256.0 * 256.0), 1.0 / 256.0, 1.0 );
+			return dot( color, bitSh ) * 1000.0;
+			
+		}
+
+		/**
+		*  cubeToUV() maps a 3D direction vector suitable for cube texture mapping to a 2D
+		*  vector suitable for 2D texture mapping. This code uses the following layout for the
+		*  2D texture:		
+		*  
+		*  xzXZ
+		*   y Y
+		*
+		*  Y - Positive y direction
+		*  y - Negative y direction
+		*  X - Positive x direction
+		*  x - Negative x direction
+		*  Z - Positive z direction
+		*  z - Negative z direction
+		*
+		*  Alternate code for a horizontal cross layout can be found here:
+		*  https://gist.github.com/tschw/da10c43c467ce8afd0c4
+		*/
```

Comment:
Also, more generally, though not a strong requirement: When having the choice, it's a good idea to opt for single-line comments, so the multi-line ones can be used to disable code. Less of an issue in GLSL (where there is `#if 0` ... `#endif`) than in JavaScript below.